javascript/javascript2/week3/main.js

```diff
@@ -0,0 +1,35 @@
+let playerS = 0;
+let playerL = 0;
+const presser = function (e) {
+  if (e.key == "l") {
+    playerL++;
+    document.getElementById("countL").innerText = playerL;
+  } else if (e.key == "s") {
+    playerS++;
+    document.getElementById("countS").innerText = playerS;
+  }
+};
+
+const inputTime = function () {
+  return document.getElementById("inputnumber").value;
+};
```

Comment:
Every time `inputTime` is called `document.getElementById("inputnumber")` has to search the DOM for this element. You can save it in a variable once to save lots of calls to `getElementById`.
```suggestion
const inputNumberElement = document.getElementById("inputnumber");
const inputTime = function () {
  return inputNumberElement.value;
};
```

Reply:
Thanks yes it is a good suggestion